docs_src/microservices/support/scheduler/EdgeX_SupportSchedulerModel.xml

```diff
@@ -0,0 +1 @@
+<mxfile host="Chrome" modified="2021-07-06T03:10:54.880Z" agent="5.0 (Macintosh; Intel Mac OS X 10_15_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36" etag="F8LD28t27LBeZS0ZePdF" version="14.7.7" type="device"><diagram id="NGmNJWxlXgWWMOgm2F2l" name="Page-1">7VpZd5s4FP41PqfzkB52249eu0zSLe606cscGRSjFhABObH760cCiUUImySm0yV+SNC92tD97nd1JQbmLNy9SEDsX2APBgND83YDcz4wjPF4RP8ywT4XOJaVCzYJ8nKRXgou0XfIhRqXbpEH01pFgnFAUFwXujiKoEtqMpAk+K5e7RoH9VFjsIENwaULgqb0E/KIz6W6ppWKlxBtfD70yOaKEIjKXJD6wMN3FZG5GJizBGOSP4W7GQzY2ol1ydstW7TFxBIYkS4NLt6cL4eu+3q3vpiai4vgy5vvwRnv5RYEW/7CryICE1rmkyZ7sRLpHQoDENHS1PVR4J2DPd6ykVMC3G+iNPVxgr7jiNAezLlOBVSdEG5YU6vVuGQtqZhJr6lkhgOcZKOZy+xH5QneRh70KrV4X7olynyOWjZYgr8VlmLDJzCl9d+JZdIK0TlIiZiiMA3TeiD1i/FAgDYRfQ7gNXu5NAYuijbnWWluswmgIKhMW9Ns23H4xJYgRAHD/Qr4OARU2rSZMABMCNxVRNyGLyAOIUn2tIrQ2nbehDvUmMPrrkQnnUYu8yvA1IVHAe4Rm6LrEjT0gePmHhgyGhii1Q3t2cePr+bZO1MUJXTV8mcPELAGKWS+DSOYAAK9vwaGQxdPZysegZDpnuVtqpoMSGoVjDy1Agk0z4zBRFWhAXPZ4hmghIUjnOG/anQuUiCjAlVqzykzMaLEMuEDEBwLLIJ1AeAEE7ZARTlm75yjVPIQTRvNMg8JwBoGU+pJm8xX5Ll2BOJBejiOTo5GsyMaRycAY7wL5ld/zyeOcTt1vyxu9sM5PtOtBhov3q9WE8+jK50+kdrPSWpFkOQwsrQmjnRHxWr9AcluAInsY/bE8FRhEB+nLawU45yvKAnVxNt1gKg5EnUrygvIVave40tFf98gjCmp3EKFLoEEoAhm7Lim+6aqDmwJ/gDFxklRgatWKITMDYref3nG7IrjwwzzGFJUg9nuDcxOA8wfFpdPrPhrsSKlxY5AMnsD0rCNFRme7s2KVblPSHwBiY8zsoLRNqy1YulUtb8/nISczmC6LwlpvWGnmSg8EdBPTUC2Vc81zaECRJYCRLbZF4ZMNf+UjPGH84LR2cYHeEFlUqMvkxrNjTbl/YBkL53/X1DTspMChCMq2nCV+C9CBCeTVYaHPghl9NsRihJqEs3Y+oTFhF4IxRoNu6FvaPeFPvXOuAKrPN37s2ml8NHutDLqZlhz+HjDbu3p2W528xIR+PqfJVj/+/nT7ZltNYwGvQ0U/kY3oT7e4AgEi1Iq+WtZ5xwzM2Tr+xUSsucWYxl0tnMNBV/AHSKfWfPnNi9d8c7Y83xXLexFIaLvW2nEileiP1Yom2Wlsp03YTcctOghEOLIW/koyhVLFJQUVsVmFRKtDp3ibeLCDmGGsuMGFi56lc6CGEdz68vdRzz8+nb9WZwNs5U/iJoEBpTeb+uXLipM8KbvME0fSl4ZiqR8X/DMc4kx8nfi7arXJFJX9lg6ipKT8vydGx1RU4B9pVrMKqQNGBfv/HDKEq6lCJjMyWuYd262WCjO0gy1E1pB1+JdqRSx1GVxDUVl9F3m/YlIK7kTZQJSR79gRReyY3cFm4XI83JHkwiNrxbt154ObHZtwHwr5cQ4aD3kpnC+zn4nik+S8U2juTuyVZujE+RMShZzmuGpSWslD/AwUzGJmvbbOaGNjLRjZKQkPkGJauJ7BP+YTf45GLiOEtARAwvZI3mqIBOBr7EEnM4spUks5Yw6sdQDOEjJ6+33xhM336Y/ZfQ/ZUYv4aY4cj56fWydgOIObRF6vz4WV8FvWmsAcRqlPSsOpjS8/gpdIl2LEBi1nG5y5YofSqgGCVFEoUta9L9BqnGaC+mDvHPyC+mTfB4xWo23Y+/mk/727QsXLvWrtWGcDf/PPEQ8X9VC87E8pBb4y31Av3mIFKKPbgwOfYvSLS85eRri6FJ4l2+nu4Z3x5AOXuVD+XsnIS37EcushwNHOzgvub5h2pJ/5DN46F5D6UBP6c6BdKfBhAq/ad0NWNIFo674mKyvdEdp6vGvzpXGCchS64EsD17mHs2i8nTrEXT5KFAIUn1CxY9Bhf2jUKGOMaZ8BiMfwHXOkS0pR9bt5/I9woPTZFosP93Oq5ffv5uL/wA=</diagram></mxfile>
```

Comment:
What is this file for?

Reply:
This is the data model source file, we drew the data model using the application https://app.diagrams.net/.